Tasks/DownloadBuildDropV0/task.json

```diff
@@ -0,0 +1,48 @@
+{
+  "id": "61F2A582-95AE-4948-B34D-A1B3C4F6A737",
+  "name": "DownloadBuildDrop",
+  "friendlyName": "Download Build Drop",
+  "description": "Download Build Drop",
+  "helpMarkDown": "",
+  "category": "Utility",
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 0,
+    "Minor": 137,
+    "Patch": 0
+  },
+  "groups": [],
+  "demands": [],
```

Comment:
you need demand min agent version for your new tasks.

Reply:
Is it something like this?

    "minimumAgentVersion": "1.91.0",

But what version should I use?

Reply:
you might want to target next agent release. do 2.138.0

Reply:
added